src/omlt/neuralnet/nn_formulation.py

```diff
@@ -6,7 +6,8 @@
 from omlt.neuralnet.layers.reduced_space import reduced_space_dense_layer
 from omlt.neuralnet.activations import (linear_activation_constraint,
                                         linear_activation_function,
-                                        bigm_relu_activation_constraint)
+                                        bigm_relu_activation_constraint,
+                                        ComplementarityReLUActivation)
```

Comment:
consider using `omlt.neuralnet.activations.linear` and `omlt.neuralnet.activations.relu` to be consistent with `omlt.neuralnet.activations.smooth` used below.

Reply:
Went a different route and imported all from omlt.neuralnet.activations.